docs/website/xamarin-analysis-doc-tool/xamarin-analysis-doc.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Xml.Linq;
+using System.IO;
+
+namespace XamarinAnalysisDoc
+{
+	class MainClass
+	{
+		static string analysis_path;
+
+		public static int Main (string [] args)
+		{
+			if (args.Length < 1) {
+				Console.WriteLine ("Usage: mono xamarin-analysis-doc.exe /path/to/Xamarin.iOS.Analysis.targets");
+				return 1;
+			}
+
+			analysis_path = args [0];
+
+			if (!File.Exists (analysis_path)) {
+				Console.WriteLine ($"Cannot find {analysis_path}");
+				return 1;
+			}
+
+			GenerateAnalysisDoc (analysis_path);
+
+			return 0;
+		}
+
+		static void GenerateAnalysisDoc (string path)
+		{
+			var section = "id:{c29b69f5-08e4-4dcc-831e-7fd692ab0886}\n";
+			section += "title:Xamarin Analysis rules\n\n";
```

Comment:
So you want Rules with a capital letter yes but I don't understand the `XIA` -> Xamarin.iOS Analysis